restapi/src/main/java/io/stargate/web/docsapi/service/query/filter/operation/BaseFilterOperation.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.stargate.web.docsapi.service.query.filter.operation;
+
+import io.stargate.db.query.Predicate;
+import java.util.Optional;
+
+/** Base predicate that can be used in the filter expression. */
+public interface BaseFilterOperation {
+
+  /**
+   * @return Returns raw value of the predicate, as user would specify in the query, f.e. <code>$eq
+   *     </code>
+   */
+  String getRawValue();
```

Comment:
Should we use an `enum` for the returned values perhaps? 

Also, how about renaming to `getOpCode` or something like that? "Raw" seems too vague, eh?

Reply:
but are we getting anything with this, yea we limit filter operations to return a fixed enum value, but in fact since we need to parse it, we are adding additional complexity here.. If the enum would bring anything else to the table it would be fine I guess.. or?

Reply:
ok connected condition provider to the enum, even better now.. Resolved!